server/mongoDB Commands/importDB.js

```diff
@@ -0,0 +1,46 @@
+const mongoose = require('mongoose');
+const fs = require('fs');
+
+const Bob = require('../models/bob');
+const Flavor = require('../models/flavor');
+const Tag = require('../models/tag');
+
+let db = mongoose.connect('mongodb://localhost/test').connection;
+
+const collectionName = process.argv[2];
+
+db.on('open', function () {
+
+  fs.readFile(collectionName + '.json', 'utf8', (err, data) => {
```

Comment:
we should be consistent across all JS in using either `function(err, data){}` or `(err, data) => {}` for function definitions - I would lean towards the former since we have a lot of that already